.github/ISSUE_TEMPLATE/release.md

```diff
@@ -3,92 +3,83 @@
 ## Step-by-step release process
 
 ### Prepare release PR
-  - [ ] Create a branch named `release-{version}`:
-    - [ ] If it's a patch release, based the new branch out of the one that
-      contains the fixes. For example, if we added some fixes to the already
-      released `release-0.12` branch, then it will be the base for our `0.12.1`
-      release.
+  - [ ] Create a branch named `release-{version}` out of `master`, not applicable for patch releases.
```

Comment:
I thought we create branch for path release as well. Is this just about naming branch differently or?

Reply:
No, we don't create branch for patch release.